### PR TITLE
12.0 mig analytic tag dimension

### PR DESCRIPTION
**Pull request by jaumeplanas**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by jaumeplanas** [comment]
Any idea why Runbot is failing?

**Comment by pedrobaeza** [comment]
It's due to 2 warnings:
```
2019-04-30 07:36:45,285 167 WARNING openerp_test odoo.models: The model account.analytic.dimension has no _description
2019-04-30 07:36:45,286 167 WARNING openerp_test odoo.models: The model analytic.dimension.line has no _description
```

**Comment by jaumeplanas** [comment]
Thanks

**Comment by jaumeplanas** [comment]
All green, when merged I'll migrate the other two dependent addons

File: analytic_tag_dimension/readme/CONFIGURE.rst
Comment (Tardo) [inline analytic_tag_dimension/readme/CONFIGURE.rst]: Update these information with the correct way to configure this module

File: analytic_tag_dimension/readme/USAGE.rst
Comment (Tardo) [inline analytic_tag_dimension/readme/USAGE.rst]: Remove this file if not really used. gen-readme script will write runbot button.

**Review by Tardo — CHANGES_REQUESTED** [review]

File: analytic_tag_dimension/readme/USAGE.rst
Comment (jaumeplanas) [reply analytic_tag_dimension/readme/USAGE.rst]: Removed

**Review by Tardo — APPROVED** [review]
LGTM, code review. Thanks :+1:

**Comment by pedrobaeza** [comment]
@jaumeplanas please NEVER resolve conflicts with a merge operation. It has been a nightmare to solve the conflicts when plaining the commit history. About the migration itself, note that on new README by fragments, you shouldn't put the title in DESCRIPTION.rst file. I have already amended this.

So let's merge this after both operations have been done.

**Pull request merged**
